src/services.js

```diff
@@ -137,4 +137,20 @@ export default {
     width: 600,
     id: ids => ids.join('/status/'),
   },
+  pinterest: {
+    regex: /https?:\/\/([^\/\?\&]*).pinterest.com\/pin\/([^\/\?\&]*)\/?$/,
+    embedUrl: 'https://assets.pinterest.com/ext/embed.html?id=<%= remote_id %>',
+    html: "<iframe scrolling='no' frameborder='no' allowtransparency='true' allowfullscreen='true' style='width: 100%; min-height: 400px; max-height: 1000px;'></iframe>",
+    id: (e) => {
+        return e[1];
+    }
+  },
```

Comment:
To be more consestant :)
```suggestion
    id: (ids) => {
        return ids[1];
    }
  },
```